src/core/documentation.h

```diff
@@ -23,6 +23,7 @@
 #define dt_DOCUMENTATION_h
 namespace dt {
 
+extern const char* doc_dt_countna;
```

Comment:
Please move this down into the block of all `doc_dt_*` constants, alphabetized.

Reply:
Ack